### ISSUE TITLE
NameError: uninitialized constant ActionController::API with `0.7.0`

### ISSUE THREAD

**Issue by paranoicsan** [amatsuda/traceroute#34]
The latest version `0.7.0` introduces the error while calling `bundle exec rake traceroute`:
```sh
NameError: uninitialized constant ActionController::API
[..]/gems/traceroute-0.7.0/lib/traceroute.rb:31:in `defined_action_methods'
[..]/gems/traceroute-0.7.0/lib/tasks/traceroute.rake:8:in `block in <top (required)>'
[..]/gems/airbrake-7.3.5/lib/airbrake/rake.rb:17:in `execute'
[..]/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
[..]/bin/ruby_executable_hooks:15:in `eval'
[..]/bin/ruby_executable_hooks:15:in `<main>'
```

```sh
ruby 2.3.3p222 (2016-11-21 revision 56859) [x86_64-darwin16]
Rails 4.2.7.1
```

Previous `0.6.2` works correctly.

**Comment by amatsuda** [comment]
@paranoicsan Released 0.7.1 gem with a fix for this issue. Thank you for the bug report!

**Comment by paranoicsan** [comment]
@amatsuda thanks for solving the issue! 🎉
